EIPS/eip-4844.md

```diff
@@ -334,7 +334,7 @@ class BlobTransactionNetworkWrapper(Container):
     # BLSFieldElement = uint256
     blobs: List[Vector[BLSFieldElement, FIELD_ELEMENTS_PER_BLOB], LIMIT_BLOBS_PER_TX]
     # KZGProof = Bytes48
-    kzg_aggregated_proof: KZGProof
+    blob_proofs: List[KZGProof, MAX_TX_WRAP_KZG_COMMITMENTS]
```

Comment:
A bit redundant to call it `blob_proofs`. Let's call it `proofs`.

Reply:
added it on the lines of `blob_kzgs`, should we also rename it to just `kzgs`?

Reply:
IMO `kzgs` is ambiguous because it's not clear if it refers to proofs or commitments.

Reply:
I think it is, but probably in a separate PR.

I was originally mistaken that you were gonna introduce another `kzgs` naming with this PR, but since it refers to something that already exists, please disregard my previous comment.

I don't have a strong opinion about `blob_kzgs` vs `kzgs`. But I guess if you go with `proofs` it also makes sense to go with `kzgs`.

Reply:
Prefer `commitments` to `kzgs`.

Reply:
updated, can rename kzgs to commitments in a followup PR

Reply:
Here is the followup PR build on top of this current branch: https://github.com/g11tech/EIPs/pull/1  (can update target to this repo once this PR merges)

Reply:
I have merged the followup changes to this PR on the request from @Inphi

Reply:
should i also rename `blob_versioned_hashes` to `versioned_hashes` in `BlobTransactionType`?